cmd/difference_test.go

```diff
@@ -24,26 +24,40 @@ import (
 var testCases = []struct {
 	pattern []string
 
-	object string
+	srcSuffix string
 
 	match bool
+
+	typ ClientURLType
```

Comment:
```suggestion
	type ClientURLType
```

Reply:
@shtripat  `type` is a keywork.. Pretty sure you can't use it here.

Reply:
ah yes. we are fine then